openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1206,3 +1214,112 @@ def validate_glusterfsd_memory_usage(self, size_limit):
                     "Failed memory used  of glusterfsd {} is greater than the"
                     " expected size {} for node/pod {}".format(
                         memory_used, size_limit, gluster_node))
+
+    def create_heketi_volumes_in_batch(
+            self, heketi_client_node, heketi_server_url, vol_count, size=1,
```

Comment:
Don't need to pass `heketi_client_node` & `heketi_server_url` parameters through functions, use variables from setup.